sources/AppBundle/Association/Model/Repository/GeneralMeetingQuestionRepository.php

```diff
@@ -3,13 +3,48 @@
 namespace AppBundle\Association\Model\Repository;
 
 use AppBundle\Association\Model\GeneralMeetingQuestion;
+use CCMBenchmark\Ting\Repository\HydratorSingleObject;
 use CCMBenchmark\Ting\Repository\Metadata;
 use CCMBenchmark\Ting\Repository\MetadataInitializer;
 use CCMBenchmark\Ting\Repository\Repository;
 use CCMBenchmark\Ting\Serializer\SerializerFactoryInterface;
 
 class GeneralMeetingQuestionRepository extends Repository implements MetadataInitializer
 {
+    public function loadNextOpenedQuestion(\DateTimeInterface $generalMeetingDate)
```

Comment:
Je trouve le nommage suivant plus adapté à ce que fait la méthode. Tu en penses quoi ?

```suggestion
    public function loadMostRecentOpenedQuestion(\DateTimeInterface $generalMeetingDate)
```

Reply:
avec le next, je m'étais dit que ça pouvait fonctionner pour le cas où par erreur on en ouvrirais deux en même temps.
vu qu'on trie par "ORDER BY afup_assemblee_generale_question.opened_at ASC" est-ce que ça ne serait pas `loadLeastRecentOpenedQuestion` (dans le cas où on a deux questions d'ouvertes ?)